i18n/ar.json

```diff
@@ -8,24 +8,25 @@
 			"أحمد",
 			"الهميان",
 			"روخو",
-			"محمد أحمد عبد الفتاح"
+		"محمد أحمد عبد الفتاح",
+			"أحمد غربية Ahmad Gharbeia <ahmad@arabdigitalexpression.org>"
 		]
 	},
-	"smw-desc": "تقرّب الويكي إلى كلٍّ من النّظم الحاسوبية  ''و''البشر ([https://www.semantic-mediawiki.org/wiki/Help:User_manual دليل المستخدم])",
+	"smw-desc": "تقرّب الويكي إلى كلٍّ من النّظم الحاسوبية  ''و'' البشر ([https://semantic-mediawiki.org/wiki/Help:User_manual دليل المستخدم])",
```

Comment:
It will be nice if you could re-add the "www" to avoid redirects. There are a couple of other affected lines in this file. In general it is preferred to translate via translatewiki.net for regular system-messages.